documentation/templates/theme/initiatives/epics/epic_template.md

```diff
@@ -1,20 +1,32 @@
 "Title of the Epic"
 
-Description: [A brief description of the epic]
+Description: Epic is a large body of work that is too big to be completed in a single iteration and needs to be broken down into smaller, more manageable pieces called "user stories."
 
-Goals: [List the high-level goals or objectives of the epic]
+Goals: To deliver a specific feature or set of features that will provide value to the end-user or customer
+        To address a particular business need or opportunity that the product or feature will fulfill.  
+        To improve the user experience or usability of the product.
+        To optimize or streamline a particular process or workflow within the product.
 
-User Stories: [List the user stories that will help achieve the goals of the epic]
+User Stories: Determine the different types of users who will interact with the product or feature being developed. This will help you to create user stories that are specific to each user's needs and expectations.
```

Comment:
You would put your user stories that go with the epic here. ChatGPT would help give ideas as to what user stories would git your epic.